splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -88,15 +88,20 @@ main() {
         do
             case "$1" in
             "-h"|"--help")
-                help 0 
+                help 0
             ;;
             "-b"|"--branch")
                 if [ -n "$2" ]; then
-                    BRANCH_TAG="$(echo $2 | cut -d'/' -f2)"
-                    HAVE_BRANCH=true
-                    shift 2
+                    if [[ `curl https://api.github.com/repos/wazuh/wazuh-api/branches` =~ "$2" ]] || [[ `curl https://api.github.com/repos/wazuh/wazuh-api/tags` =~ "$2" ]]; then
```

Comment:
This check isn't necessary. `git` automatically checks this while cloning the branch.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20